src/main/java/com/bringframework/annotations/Component.java

```diff
@@ -0,0 +1,23 @@
+package com.bringframework.annotations;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation used to mark a class as a bean that can be managed by an IoC container. The class will be instantiated and
+ * configured by the container, and its lifecycle will be managed.
+ */
+@Target({TYPE})
+@Retention(RUNTIME)
+public @interface Component {
+
+  /**
+   * The name of the bean. If no value is specified, the name of the class is used.
+   *
+   * @return the name of the bean
+   */
+  String value() default "";
+}
```

Comment:
GitHub shows warning that this file don't have new line at the end

Reply:
Fixed